// excessão são importantes, pois através da pilha de excessões você consegue saber exatament em qual função ou parte do código o erro está acontecendo
fun main() {
    throw Exception("Hi There!")
}
